helm/alfresco-process-infrastructure/setup_acs.sh

```diff
@@ -2,7 +2,7 @@
 
 source $(dirname "$0")/functions_acs.sh
 
-create_user "service-account-storage-service"
+create_user "service-account-storage-service" "sass@activiti.org" $(generate_alfpwd)
```

Comment:
use service-account-storage-service@example.com as password as that is an official dummy domain to use

Reply:
done